src/Kros.AspNetCore/Extensions/HealthChecksBuilderExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Kros.Identity.Extensions;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Linq;
+
+namespace Kros.AspNetCore.Extensions
+{
+    public static class HealthChecksBuilderExtensions
+    {
+        public static IHealthChecksBuilder AddIdentityServerHealthChecks(this IHealthChecksBuilder healthChecksBuilder, IConfiguration configuration)
+        {
+            var identityServerOptions = configuration.GetSection("IdentityServerHandlers").Get<IdentityServerOptions[]>().ToList();
```

Comment:
We have generic extension method `GetSection<T>()` for getting options from configuration.
It may look like this: `configuration.GetSection<IdentityServerOptions>()`.